domain/robstoll-lib/atrium-domain-robstoll-lib-common/src/main/kotlin/ch/tutteli/atrium/domain/robstoll/lib/creating/anyAssertions.kt

```diff
@@ -34,20 +34,16 @@ fun <T> _isNotSame(subjectProvider: SubjectProvider<T>, expected: T) =
     ExpectImpl.builder.createDescriptive(subjectProvider, IS_NOT_SAME, expected) { it !== expected }
 
 fun <T> _isNotIn(subjectProvider: SubjectProvider<T>, expected: List<T>) =
-    ExpectImpl.builder.createDescriptive(subjectProvider, IS_NOT_IN, expected) { it !== expected }
+    ExpectImpl.builder.createDescriptive(subjectProvider, IS_NOT_IN, expected) { it !== expected}
 
 private fun <T> _isNoneOf(
     expected: List<T>,
     warning: DescriptionAnyAssertion
 ): AssertionGroup {
-    val assertions = expected.map{ ExpectImpl.builder.representationOnly}
-    val additionalEntries = ExpectImpl.builder.list
+    val assertion = expected.map{ ExpectImpl.builder.representationOnly.failing.withRepresentation(IS_NONE_OF).build()}
```

Comment:
I suggest add the specs and make an assertion fail on purpose. You will see, that this implementation is not what we want. Almost but not as it should be.
Also, rename this function to _isNotIn and remove the one above